applications/logbook/logbook-plugins/org.csstudio.logbook.olog/src/org/csstudio/logbook/olog/OlogLogbookClient.java

```diff
@@ -29,8 +29,8 @@
 import org.csstudio.logbook.Property;
 import org.csstudio.logbook.Tag;
 import org.csstudio.logbook.util.LogEntrySearchUtil;
-import org.epics.util.time.TimeInterval;
-import org.epics.util.time.TimeParser;
+import org.diirt.util.time.TimeInterval;
```

Comment:
@shroffk changes I made to logbook.olog.  I just noticed I have the version as 3.0.1 on org.diirt.util.time ... we should removed that.